### ISSUE TITLE
arduino sketch fails to upload from the latest source

### ISSUE THREAD

**Issue by moonbit5** [eolvera85/PS2toXBOX#2]
hi do you have a .hex + .eep for the latest version? 

the arduino sketch fails to upload from the latest source

**Comment by eolvera85** [comment]
Hi, have you already tried to compile the version without the LUFA libraries?

I share with you the Hex, epp of both versions. 
[PS2toXbox.zip](https://github.com/eolvera85/PS2toXBOX/files/13832570/PS2toXbox.zip)

If you are trying to make the adapter for bluetooth controls with BlueRetro, I recommend the definitive version that we have on David's channel, (in Spanish) https://youtu.be/r4PG4MqMDyg?si=VV54DnEespdMiB1-

**Comment by moonbit5** [comment]
thank you. I was attempting to update blueretro 1.9 https://github.com/konwektor/BlueRetro/tree/master

and assumed I needed this latest version, however it doesn't work as you probably know. 

guess I need to update something in the blueretro 1.9 config somewhere? 

[this is the wiring diagram I used ](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEgS_sfG9mG6dgQ57U1vkJRqyjWiTWYb-8RPiz4Gm9rif06wDbVTv3rgKvLRi9gEG1uhWFPm9jp7mjB3fiBRwYNiXY5Yvl10piNfWPTrt3o1q9wxd_hkA-NB3zQNBgTYwVy2Q98zFLx0X10ysvvEhFl9jGSDrGJbHuLeh0VXJ6IWK1NIdmAt_VAqZLKSMw/s4680/DIAGRAMA_XBOX_OG_NEW.png)

**Comment by eolvera85** [comment]
I'll try to explain.

- I don't know if what Konwektor is doing is functional, I haven't tested it.
- What we did for our part was to stabilize the basic functionality with new generation and more commercial controllers (Xbox Series S/X and PS5/PS4).
- Most likely with other controllers they will crash or present problems.
- Konwektor contacted us to request the source code of the changes we made. (I don't know if he is integrating it into his changes).
- We are not looking to take the project to the next level (Internal Installation, compatibility with more controllers)
- The compiled binaries of our version are in this [link](https://davidxg.blogspot.com/2023/11/blueogx.html)
